tests/huggingface/sagemaker_dlc_test.py

```diff
@@ -49,7 +53,8 @@ def run_test(args):
         predictor = model.deploy(instance_type=args.instance_type,
                                  initial_instance_count=1,
                                  endpoint_name=endpoint_name,
-                                 container_startup_health_check_timeout=1800
+                                 container_startup_health_check_timeout=1800,
+                                 volume_size=256,
```

Comment:
Do we not need to increase `model_data_download_timeout_in_seconds`?

Reply:
Also a reminder that this will need to be reflected in huggingface.co codes for any inf2 enabled models.

Reply:
The model should be deployed in less than 900 seconds actually.

Reply:
Sorry i meant the model data download timeout not model deployment timeout. But the tests seem green so it should be ok.

Reply:
Could you please ack on :
> Also a reminder that this will need to be reflected in huggingface.co codes for any inf2 enabled models.